server/src/main/java/org/candlepin/dto/StandardTranslator.java

```diff
@@ -69,6 +72,8 @@ public StandardTranslator() {
             new UpstreamConsumerTranslator(), UpstreamConsumer.class, UpstreamConsumerDTO.class);
         this.registerTranslator(
             new ActivationKeyTranslator(), ActivationKey.class, ActivationKeyDTO.class);
+        this.registerTranslator(
```

Comment:
For the sake of organization, this doesn't belong with the API translators. Throw a new header in for the event translators.